src/Essential.Diagnostics.RollingFileTraceListener/Diagnostics/RollingFileTraceListener.cs

```diff
@@ -37,10 +37,16 @@ public class RollingFileTraceListener : TraceListenerBase
             { 
                 "template", "Template", 
                 "convertWriteToEvent", "ConvertWriteToEvent",
+                "createSubdirectories", "CreateSubdirectories"
```

Comment:
Just call it createDirectories / CreateDirectories, as it will actually create all directories (whether they are subdirectories or top level directories). Most of the time it will be used to only create subdirectories, but it will actually create both, so should be named accordingly.